nerd/conf/conf.go

```diff
@@ -42,7 +42,7 @@ func Defaults() *Config {
 		Auth: AuthConfig{
 			APIEndpoint:      "https://auth.nerdalize.com/v1/",
 			OAuthLocalServer: "localhost:9876",
-			OAuthSuccessURL:  "https://cloud.nerdalize.com",
+			OAuthSuccessURL:  "https://auth.nerdalic.eom/do/login_complete/?client=CLI",
```

Comment:
Is this the right url ? Shouldn't it be `https://auth.nerdalize.com/do/login_complete/?client=CLI` ?